src/lib/drawer/Drawer.svelte

```diff
@@ -0,0 +1,45 @@
+<script lang="ts">
+	import type { SvelteHTMLElements } from 'svelte/elements';
+	import { twMerge } from 'tailwind-merge';
+	import Button from '$lib/Button.svelte';
+	import { fly } from 'svelte/transition';
+
+	export let open = false;
+
+	function closeDrawer() {
+		open = false;
+	}
+
+	type $$Props = SvelteHTMLElements['div'];
+
+	let className: $$Props['class'] = undefined;
+	export { className as class };
+</script>
+
+{#if open}
+	<div
+		tabindex="-1"
+		in:fly={{ x: 500, duration: 300 }}
+		out:fly={{ x: 500, duration: 300 }}
```

Comment:
As they have the same properties, this can be simplified. Also I _think_ you can use '100%' as the x value so we're not hard coding the width
```suggestion
		transition:fly={{ x: '100%', duration: 300 }}
```